ggml.c

```diff
@@ -17200,7 +17230,13 @@ struct ggml_cplan ggml_graph_plan(struct ggml_cgraph * cgraph, int n_threads) {
                     }
                 } break;
             case GGML_OP_SILU_BACK:
+                {
+                    n_tasks = n_threads;
+                } break;
```

Comment:
You could move GGML_OP_SILU_BACK to after GGML_OP_MUL to avoid the duplicate code.